src/status_im2/common/parallax/view.cljs

```diff
@@ -0,0 +1,69 @@
+(ns status-im2.common.parallax.view
+  (:require [react-native.reanimated :as reanimated]
+            [reagent.core :as reagent]
+            [react-native.core :as rn]
+            [utils.worklets.parallax :as worklets.parallax]
+            ["react-native-transparent-video" :default TV]
+            [react-native.safe-area :as safe-area]
+            [react-native.platform :as platform]))
+
+(def transparent-video (reagent/adapt-react-class TV))
```

Comment:
Maybe move to a separate namespace? It could be `react-native.transparent-video`

Reply:
sure!